### ISSUE TITLE
Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads

### ISSUE THREAD

**Issue by TheMissingPort** [DIGITALCRIMINALS/OnlyFans#57]
Every time I try to scrape I get this error after it logs me in. Everything is up to date. Not sure why 

`INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads`

basically it does this in a loop and I have to quit terminal

Any ideas? Thanks

Edit: Weird, it only happens if I move the folder locations?

**Comment by americanseeder1865** [comment]
> Edit: Weird, it only happens if I move the folder locations?

Move which folder? This might be doing something

>INFO [alembic.runtime.migration] Context impl SQLiteImpl. INFO [alembic.runtime.migration] Will assume non-transactional DDL. Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads

So this is a database issue. This will be fun to troubleshoot/debugging. Can you answer me some questions?

- [ ] Are you using Python 3.10?
- [ ] What operating system are you using? If you are using docker, please let me know here.
- [ ] Are you running the scraper in a virtual environment?
